sdk/tvOS/OoyalaTVSkinSDK/Widgets/OOOoyalaTVClosedCaptionsView.h

```diff
@@ -0,0 +1,15 @@
+#import <UIKit/UIKit.h>
+
+@class OOCaption;
+@class OOClosedCaptionsStyle;
+
+@interface OOOoyalaTVClosedCaptionsView : UIView {
+    OOCaption *caption;
```

Comment:
By defining these variables, you are effectively defining the ivars for the same properties. I don't think you need to define these since they will be created by default by their respective properties below.